UPDATE.md

```diff
@@ -50,13 +52,9 @@ See here for more information:
 
 ### Execute the media update
 All you need to do now is:
+
 ```
-drush updb
-```
-Then you will be informed that all your media_entiy code has to be
-media in core compatible now. If you are sure that you are ready for the
-migration, call the drush command again.
-```
+drush cr
```

Comment:
I'm not sure about this - because `drush cr` could fail with new code base without database updated. It would be better to executed `drush cr` with old code base and then just `drush updb` with new code.